packages/public/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUDAddDamagePreview/CrosshairPreview/CrosshairPreview.tsx

```diff
@@ -0,0 +1,40 @@
.25C55.2426 101.25 56.25 100.243 56.25 99H55.75C55.75 99.9665 54.9665 100.75 54 100.75V101.25ZM51.75 99C51.75 100.243 52.7574 101.25 54 101.25V100.75C53.0335 100.75 52.25 99.9665 52.25 99H51.75ZM51.75 83V99H52.25V83H51.75ZM6.25 53C6.25 52.0335 7.0335 51.25 8 51.25V50.75C6.75736 50.75 5.75 51.7574 5.75 53H6.25ZM8 54.75C7.0335 54.75 6.25 53.9665 6.25 53H5.75C5.75 54.2426 6.75736 55.25 8 55.25V54.75ZM24 54.75H8V55.25H24V54.75ZM25.75 53C25.75 53.9665 24.9665 54.75 24 54.75V55.25C25.2426 55.25 26.25 54.2426 26.25 53H25.75ZM24 51.25C24.9665 51.25 25.75 52.0335 25.75 53H26.25C26.25 51.7574 25.2426 50.75 24 50.75V51.25ZM8 51.25H24V50.75H8V51.25ZM84 50.75C82.7574 50.75 81.75 51.7574 81.75 53H82.25C82.25 52.0335 83.0335 51.25 84 51.25V50.75ZM100 50.75H84V51.25H100V50.75ZM102.25 53C102.25 51.7574 101.243 50.75 100 50.75V51.25C100.966 51.25 101.75 52.0335 101.75 53H102.25ZM100 55.25C101.243 55.25 102.25 54.2426 102.25 53H101.75C101.75 53.9665 100.966 54.75 100 54.75V55.25ZM84 55.25H100V54.75H84V55.25ZM81.75 53C81.75 54.2426 82.7574 55.25 84 55.25V54.75C83.0335 54.75 82.25 53.9665 82.25 53H81.75ZM52.25 53C52.25 52.0335 53.0335 51.25 54 51.25V50.75C52.7574 50.75 51.75 51.7574 51.75 53H52.25ZM54 54.75C53.0335 54.75 52.25 53.9665 52.25 53H51.75C51.75 54.2426 52.7574 55.25 54 55.25V54.75ZM55.75 53C55.75 53.9665 54.9665 54.75 54 54.75V55.25C55.2426 55.25 56.25 54.2426 56.25 53H55.75ZM54 51.25C54.9665 51.25 55.75 52.0335 55.75 53H56.25C56.25 51.7574 55.2426 50.75 54 50.75V51.25Z" fill="#795900" mask="url(#path-1-outside-1_1083_28985)"/> </g> <defs> <filter id="filter0_ddd_1083_28985" x="0.75" y="0.75" width="106.5" height="107.5" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"> <feFlood flood-opacity="0" result="BackgroundImageFix"/> <feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/> <feOffset dy="1"/> <feGaussianBlur stdDeviation="2.5"/> <feColorMatrix type="matrix" values="0 0 0 0 0.517647 0 0 0 0 0.517647 0 0 0 0 0.517647 0 0 0 0.2 0"/> <feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow_1083_28985"/> <feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/> <feOffset dy="3"/> <feGaussianBlur stdDeviation="2"/> <feColorMatrix type="matrix" values="0 0 0 0 0.517647 0 0 0 0 0.517647 0 0 0 0 0.517647 0 0 0 0.12 0"/> <feBlend mode="normal" in2="effect1_dropShadow_1083_28985" result="effect2_dropShadow_1083_28985"/> <feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/> <feOffset dy="2"/> <feGaussianBlur stdDeviation="2"/> <feColorMatrix type="matrix" values="0 0 0 0 0.517647 0 0 0 0 0.517647 0 0 0 0 0.517647 0 0 0 0.14 0"/> <feBlend mode="normal" in2="effect2_dropShadow_1083_28985" result="effect3_dropShadow_1083_28985"/> <feBlend mode="normal" in="SourceGraphic" in2="effect3_dropShadow_1083_28985" result="shape"/> </filter> </defs> </svg>';
```

Comment:
I forgot to mention one thing about SVGs. In order to make sure that your SVG is properly formatted, minified and cleaned, you need to pass it through SVGO. SVGO is a tool that cleans the code inside SVG files and removes useless parts and fixes issues (it acts as a linter for SVG files). We already have SVGO installed and configured in the project (for example in the Sights package). You can simply create an SVG file and place your SVG code inside it. Then, you can place this file in a package that has SVGO installed (for instance, the Sights package), then place yourself inside this package and run `yarn run svgo`. Your file should be cleaned and then you can place the code back into your component in React as you did